.travis.yml

```diff
@@ -14,6 +14,12 @@ addons:
   postgresql: 9.4
 
 rvm:
+  - 2.3.0
```

Comment:
It's probably not necessary to test every single patch release version via travis ... that will be a lot of builds for every PR/branch.

Specifying either just a) the minor release versions we want to support, or b) the point releases within the minor release we want to support - is probably fine.